Base/usr/share/man/man1/su.md

```diff
@@ -1,41 +1,42 @@
 ## Name
 
-su - run a command with substitute user
+su - switch to another user
 
 ## Synopsis
 
-```
-$ su [user]
+```**sh
+$ su [--help] [--version] <user>
 ```
 
 ## Description
 
-**su** allows commands to be run with a substitute user.
-
-When called with no user-specified, **su** defaults to running an interactive shell as *root*. 
+`su` switch to another user.
 
-## Options
+When called with no user-specified, `su` defaults to switch to the *root* user. 
+**NOTE**
 
-- **--help**
+* Need to enter the password if the user switch to has one.
```

Comment:
What's up with this? That's very weird formatting and it should probably be in the general description.